library/include/borealis/tab_frame.hpp

```diff
@@ -51,9 +54,10 @@ class TabFrame : public AppletFrame
   private:
     Sidebar* sidebar;
     BoxLayout* layout;
-    View* rightPane = nullptr;
+    LayerView* rightPane = nullptr;
+    std::map<u_int64_t, int> panelMap;
```

Comment:
Why the need for such a map? Can't you just use the int id straight from LayerView?